# 单车控制量
float32 ax      # 纵向加速度 m/s^2
float32 delta   # 前轮转角 deg
